tests/components/knx/test_binary_sensor.py

```diff
@@ -42,7 +37,7 @@ async def test_binary_sensor_entity_category(hass: HomeAssistant, knx: KNXTestKi
 
     registry = await async_get_entity_registry(hass)
     entity = registry.async_get("binary_sensor.test_normal")
-    assert entity.entity_category == ENTITY_CATEGORY_DIAGNOSTIC
+    assert entity.entity_category == EntityCategory.DIAGNOSTIC
```

Comment:
Please use `is` for `entity.entity_category` comparison

Reply:
Agreed, done.